Package.swift

```diff
@@ -41,7 +41,7 @@ let package = Package(
         Target(
             /** Fetches Packages and their dependencies */
             name: "Get",
-            dependencies: ["PackageDescription", "PackageType"]),
+            dependencies: ["PackageDescription", "PackageType", "Multitool", "Utility"]),
```

Comment:
I'm really sorry that I realized the boostrap failed.
I should make an effort to prevent it, but I don't know what I should do.

- Test Case '-[FunctionalTestSuite.MiscellaneousTestCase testExternalDependencyEdges1]' failed
- Test Case '-[FunctionalTestSuite.MiscellaneousTestCase testExternalDependencyEdges2]' failed